cloudoperators/juno
# chore(infra): integrate husky to the development process

# Summary
We make our life as reviewer easier and as developers more painful :) from now on all commits should follow commit lint scopes and kebab cases for more info check docs 👍 
# Changes Made



- pre-commit hooks
- pre  push hooks
- update docs

# Related Issues
- #195 

# Testing Instructions
try to add a bad commit like chore(WRONG): WHAtever 
it should throw you an error

# Checklist
